apps/cli/src/constants.ts

```diff
@@ -1,6 +1,24 @@
 import { join } from "node:path";
 import envPaths from "env-paths";
 
-const APP_CONFIG = envPaths("webstudio").config;
+const GLOBAL_APP_CONFIG = envPaths("webstudio").config;
 const GLOBAL_CONFIG_FILE_NAME = "webstudio-config.json";
-export const GLOBAL_CONFIG_FILE = join(APP_CONFIG, GLOBAL_CONFIG_FILE_NAME);
+export const GLOBAL_CONFIG_FILE = join(
+  GLOBAL_APP_CONFIG,
+  GLOBAL_CONFIG_FILE_NAME
+);
+
+export const PROJECT_NAME = "webstudio-project";
+export const LOCAL_CONFIG_FILE_NAME = ".webstudio/config.json";
```

Comment:
join(cwd(), ".webstudio/config.json")

PROJECT_NAME not needed at all

Reply:
Changing it, But that means, users should create a empty folder and then run these commands inside of it right. Unlike cli tooks like `cra` etc which creates the empty project and clones everything inside of it.

Reply:
See, we want single ws project per package.json. if user want multiple projects then it can be monorepo.